feature-home/src/main/java/cm/aptoide/pt/feature_home/domain/BundlesUseCase.kt

```diff
@@ -22,7 +22,12 @@ class BundlesUseCase @Inject constructor(
           bundle.type == WidgetType.APPC_BANNER
         }?.let { bonusBundle ->
           BonusData.setBonusData(bonusBundle.title, bonusBundle.tag)
-        }?: urlsCache.putAll(mapOf(BonusData.currentData))
+        } ?: urlsCache.putAll(
+          mapOf(
+            "bonus-banner-more" to "https://ws75.aptoide.com/api/7.20240701/listApps/store_id=3613731/group_id=15614123/order=rand"
+          )
+        )
+        urlsCache.putAll(mapOf(BonusData.currentData))
```

Comment:
The last line is not needed anymore, otherwise it is associating a title to a tag inside the urlsCache, which does not make sense